### ISSUE TITLE
Headless Chrome workaround raises error

### ISSUE THREAD

**Issue by erikreedstrom** [teamcapybara/capybara#1877]
## Meta
Capybara Version:
2.14.1

Driver Information (and browser if relevant):
chromedriver=2.30.477691
chrome=59.0.3071.86
platform=Linux 3.13.0-119-generic x86_64

## Expected Behavior
When running test on headless chrome, accept modal functionality works.

## Actual Behavior
When running test on headless chrome, work around for modals raises error.

```
Selenium::WebDriver::Error::UnknownError:
       unknown error: Cannot read property 'current_modal_status' of undefined
```

## Steps to reproduce

Setup chrome options as so:
```
desired_capabilities: {
  chrome_options: {
    'args' => %w(headless no-sandbox disable-gpu)
  }
}
```

Run test including step:
```ruby
accept_alert do
  # some action
end
```

**Note:** This behavior is witnessed on Circle CI (with latest chrome installed). It passes locally in OS X when running headless.

**Comment by twalpole** [comment]
This works in the travis tests -- what action are you doing in the "accept_alert"

**Comment by twalpole** [comment]
Oh -- you must be performing an action that triggers a page change.  Working on a fix.

**Comment by twalpole** [comment]
Should be fixed in 2.14.2
